ethereum/permissioning/src/test/java/tech/pegasys/pantheon/ethereum/permissioning/node/NodePermissioningControllerTest.java

```diff
@@ -57,4 +67,62 @@ public void peerDiscoveryCallbackShouldBeDelegatedToSyncStatusNodePermissioningP
 
     verify(syncStatusNodePermissioningProvider).setHasReachedSyncCallback(any(Runnable.class));
   }
+
+  @Test
+  public void whenInSyncWeShouldDelegateToLocalConfigNodePermissioningProvider() {
```

Comment:
This test is wrong. When using the local config permissioning only, the SyncStatusPermissioningProvider should not be used.

Reply:
renamed and corrected the test - check behaviour when no SyncStatusProvider